proto/viam/app/mltraining/v1/ml_training.proto

```diff
@@ -69,14 +67,3 @@ message TrainingJobMetadata {
   string user_email = 6 [(tagger.v1.tags) = "bson:\"user_email\" json:\"user_email\""];
   string id = 7 [(tagger.v1.tags) = "bson:\"_id\" json:\"id,omitempty\""];
 }
-
-message TrainingJob {
```

Comment:
Why are we eliminating this? We just use a Go struct instead?

Reply:
Yup! Instead, we have a `dbTrainingJob` go struct that we insert into the `jobs` db. 

In this ticket [here](https://viam.atlassian.net/jira/software/c/projects/DATA/boards/40?modal=detail&selectedIssue=DATA-1071), Aaron mentions this is only used internally so no need to exposed in API.